### PR TITLE
add DEGEN

### PR DESCRIPTION
**Pull request by wschwab**
added the DEGEN index from Indexed

### CODE REVIEW

**Review by LefterisJP — COMMENTED** [review]
Hey @wschwab thanks a lot for the PR.

We can add this but at the moment and until v1.16.0 we are in a point where we can't automatically push more assets to users automatically due to #2550 and #2551

So until then this will remain in develop.

For tests to pass you also need to edit the all_assets.meta with the md5 checksum of the edited file and also the corresponding unit test.

Read this to see how: https://rotki.readthedocs.io/en/latest/contribute.html#adding-new-assets-to-rotki

**important**: Due to the limitations I mentioned above please do not bump the version in all_assets.meta

Also please add a changelog entry for this new token, as per the other tokens already there:https://github.com/rotki/rotki/blob/develop/docs/changelog.rst

**Comment by wschwab** [comment]
oops! I missed the last paragraph in the "adding a new asset" docs. I'll try to get to this on Sunday.

File: rotkehlchen/data/all_assets.json [hunk 1]
```diff
@@ -3591,6 +3591,15 @@
         "symbol": "DEFI5",
         "type": "ethereum token"
     },
+    "DEGEN": {
+        "coingecko": "degen-index",
+        "ethereum_address": "0x126c121f99e1e211df2e5f8de2d96fa36647c855",
+        "ethereum_token_decimals": 18,
+        "name": "DEGEN Index",
+        "started": 1614587427,
```
[the remark this answers is not in the document]
Comment (wschwab) [reply rotkehlchen/data/all_assets.json:3599]: Interesting - I specifically tried to keep everything in UTC, wonder how that crept in, but I'll fix it when I get the chance

**Comment by isidorosp** [comment]
Since you've already gone ahead and done this, can you also add the ORCL-5 index (also from Indexed Finance? token: https://www.coingecko.com/en/coins/oracle-top-5-index). I would also suggest also enforcing similar syntax/naming for all of the indexed-finance tokens we have added already (DEFI5, CC10) for consistency.

(see here):
https://github.com/rotki/rotki/pull/2300/files
```
    "CC10": {
        "coingecko": "cryptocurrency-top-10-tokens-index",
        "cryptocompare": "",
        "ethereum_address": "0x17aC188e09A7890a1844E5E65471fE8b0CcFadF3",
        "ethereum_token_decimals": 18,
        "name": "Indexed Finance - CC10",
        "started": 1607251191,
        "symbol": "CC10",
        "type": "ethereum token"
    },
```

**Comment by wschwab** [comment]
> Since you've already gone ahead and done this, can you also add the ORCL-5 index (also from Indexed Finance? token: https://www.coingecko.com/en/coins/oracle-top-5-index). I would also suggest also enforcing similar syntax/naming for all of the indexed-finance tokens we have added already (DEFI5, CC10) for consistency.

That shouldn't be a problem - this PR was meant to kind of be my learning curve first attempt, once I get in the groove, throwing in more tokens shouldn't be hard. Am I right that the only change you're proposing to the Degen Index entry is changing the `name` to `"Indexed Finanace - DEGEN"`?

**Comment by isidorosp** [comment]
> > Since you've already gone ahead and done this, can you also add the ORCL-5 index (also from Indexed Finance? token: https://www.coingecko.com/en/coins/oracle-top-5-index). I would also suggest also enforcing similar syntax/naming for all of the indexed-finance tokens we have added already (DEFI5, CC10) for consistency.
> 
> That shouldn't be a problem - this PR was meant to kind of be my learning curve first attempt, once I get in the groove, throwing in more tokens shouldn't be hard. Am I right that the only change you're proposing to the Degen Index entry is changing the `name` to `"Indexed Finanace - DEGEN"`?

Correct (sans the spelling typo) ;)

File: rotkehlchen/data/all_assets.json [hunk 2]
```diff
@@ -3591,6 +3591,15 @@
         "symbol": "DEFI5",
         "type": "ethereum token"
     },
+    "DEGEN": {
+        "coingecko": "degen-index",
+        "ethereum_address": "0x126c121f99e1e211df2e5f8de2d96fa36647c855",
+        "ethereum_token_decimals": 18,
+        "name": "DEGEN Index",
```
Comment (wschwab) [inline rotkehlchen/data/all_assets.json:3598]: should be changed to `"Indexed Finance - DEGEN"`

**Comment by wschwab** [comment]
lol, apparently a new version was released, I'll need to redo some stuff

**Comment by LefterisJP** [comment]
The requested assets are now added in the new method of assets with a DB update here: https://github.com/rotki/assets/blob/develop/updates/1/updates.sql

**Pull request closed without merging**